packages/indexer-common/src/errors.ts

```diff
@@ -144,12 +144,13 @@ export const INDEXER_ERROR_MESSAGES: Record<IndexerErrorCode, string> = {
   IE060: 'Failed to allocate: Already allocating to the subgraph deployment',
   IE061: 'Failed to allocate: Invalid allocation amount provided.',
   IE062: 'Did not receive tx receipt, not authorized or network paused.',
-  IE063: 'No active allocation with provided id found.',
-  IE064: 'Failed to unallocate: Allocations cannot be closed in this epoch.',
-  IE065: 'Failed to unallocate: Allocation has already been closed.',
-  IE066: 'Failed to allocate: allocation ID already exist on chain',
+  IE063: 'No active allocation with provided id found',
```

Comment:
We might as well remove the `.` for IE061 and IE062?

Reply:
Oops, I missed a few .'s.  I'll remove.